ex_10.js

```diff
@@ -0,0 +1,10 @@
+const massId = [3, 1, 2]
+
+const summPrice = (products, massId) =>
+  products.reduce((accum, { id, price, currency }) => {
+    if (!accum[currency] && massId.includes(id))
```

Comment:
const hasProduct =  massId.includes(id);  and use it in the next conditions
 why? because massId.includes(id) - 2 times